deploy/kubernetes/complete-demo.yaml

```diff
@@ -592,17 +556,14 @@ metadata:
         prometheus.io/scrape: 'true'
   labels:
     name: queue-master
-  annotations:
-    prometheus.io/path: "/prometheus"
   namespace: sock-shop
 spec:
   ports:
     # the port that this service should serve on
   - port: 80
     targetPort: 80
   selector:
-    name: queue-master
----
+    name: queue-master---
```

Comment:
this selector name changed on the generation of the manifest and it causes that the service is not created!

Reply:
I modified the command to create the complete-demo file so this doesn't happen again